modules/monitoring-setting-up-alert-routing-for-user-defined-projects.adoc

```diff
@@ -1,13 +1,15 @@
 // Module included in the following assemblies:
 //
-// * monitoring/enabling-alert-routing-for-user-defined-projects.adoc
+// * monitoring/configuring-alerts-for-user-defined-projects.adoc
 
 :_content-type: CONCEPT
-[id="understanding-alert-routing-for-user-defined-projects_{context}"]
-= Understanding alert routing for user-defined projects
+[id="setting-up-alert-routing-for-user-defined-projects_{context}"]
+= Setting up alert routing for user-defined projects
 
 [role="_abstract"]
-As a cluster administrator, you can enable alert routing for user-defined projects. After doing so, you can allow users to configure alert routing for their user-defined projects. Users can then create and configure user-defined alert routing by creating or editing the `AlertmanagerConfig` objects.
+As a cluster administrator, you can set up alert routing for user-defined projects. 
+After doing so, you can allow users to configure alert routing for their user-defined projects using an Alertmanager instance dedicated to user workload monitoring. 
+Users can then create and configure user-defined alert routing by creating or editing the `AlertmanagerConfig` objects for their user-defined projects without the help of an administrator.
```

Comment:
I think that we need an update below too. There are 2 possible configurations:
1. No dedicated Alertmanager for user-defined alerts -> the admin should add `alertmanagerMain: {enableUserAlertmanagerConfig: true}` to the `cluster-monitoring-config` configmap in the `openshift-monitoring` namespace.
2. Dedicated Alertmanager for user-defined alerts -> the admin should add `alertmanager: {enabled: true, enableUserAlertmanagerConfig: true}` to the `user-workload-monitoring-config` configmap in the `openshift-user-workload-monitoring` namespace.

Reply:
for the second configuration, from https://github.com/openshift/cluster-monitoring-operator/blob/release-4.11/pkg/manifests/config.go#L578-L587
```
// AlertmanagerUserWorkloadConfig defines the Alertmanager configuration for UWM.
type AlertmanagerUserWorkloadConfig struct {
	Enabled                  bool                                 `json:"enabled"`
	EnableAlertmanagerConfig bool                                 `json:"enableAlertmanagerConfig"`
	LogLevel                 string                               `json:"logLevel"`
	NodeSelector             map[string]string                    `json:"nodeSelector"`
	Tolerations              []v1.Toleration                      `json:"tolerations"`
	Resources                *v1.ResourceRequirements             `json:"resources"`
	VolumeClaimTemplate      *monv1.EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate"`
}
```
`the admin should add alertmanager: {enabled: true, enableUserAlertmanagerConfig: true}`
should be
`the admin should add alertmanager: {enabled: true, enableAlertmanagerConfig: true}`
verified in https://bugzilla.redhat.com/show_bug.cgi?id=2099937#c1

Reply:
I added an extra procedure for enabling user-defined alerts for the default platform Alertmanager instance.